src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -626,6 +646,24 @@ def app_set_deployment(cmd, client, resource_group, service, name, deployment):
     return client.apps.update(resource_group, service, name, app_resource)
 
 
+def app_unset_deployment(cmd, client, resource_group, service, name):
+    active_deployment_name = client.apps.get(
+        resource_group, service, name).properties.active_deployment_name
+    if not active_deployment_name:
+        raise CLIError(NO_PRODUCTION_DEPLOYMENT_SET_ERROR)
+
+    properties = models.AppResourceProperties(active_deployment_name="")
```

Comment:
Suggest add some comment here as String.Empty means unset action is the design to not confuse later coder